docs/contributing/dev-setup.md

```diff
@@ -24,7 +24,7 @@ The easiest & safest way to develop & test TLJH is with [Docker](https://www.doc
      --detach \
      --name=tljh-dev \
      --publish 12000:80 \
-     --mount type=bind,source=$(pwd),target=/srv/src \
+     --mount type=bind,source="$(pwd)",target=/srv/src \
```

Comment:
Same! It has been tested. Command continues to run correctly when the `source` path has no spaces as before, and now works if there are spaces since `"$(pwd)" gets expanded and quoted.